addon/index.js

```diff
@@ -48,6 +48,7 @@ const URL = require('sdk/url').URL;
 const IdeaTown = require('idea-town');
```

Comment:
Re: ESLint errors:
```
48:7  error    "IdeaTown" is defined but never used  no-unused-vars
```

We can probably remove this import since we removed lines 66-68 below.